@xen-orchestra/web-core/lib/components/LegendTitle.vue

```diff
@@ -0,0 +1,39 @@
+<!-- v1.0 -->
+<template>
+  <div class="legend-title typo c3-semi-bold">
+    {{ title }}
```

Comment:
Prefer using a `slot` over a `prop` for main content.

```suggestion
    <slot />
```